integration-tests/java/README.md

```diff
@@ -28,4 +28,42 @@ You can run tests from the top level directory with the following command.
 
 ```bash
 $ ./gradlew integration-tests:java:integrationTest
-``` 
+```
+
+# How to initiate a new project from this example
+This section explains how you can write your Java clients based on this example.
+
+### Set up a new directory
+Create a new directory, and copy entire code Java integration tests to there.
+
+```bash
+$ mkdir my-client
+$ cd my-client
+$ copy -R PATH_TO_STELLARSTATION_API/integration-tests/java/* ./
+```
+
+### Replace a dependency in build.gradle
+In order to run those test outside of stellarstation-api, you need to replace the dependency
```

Comment:
In order to run these tests in your own copy of the source code, you need to update the dependency on stellarstation-api from an internal reference to the external one.